src/views/ManageTeamHealth.vue

```diff
@@ -139,39 +138,72 @@ export default {
       return 0;
     },
 
+    /**
+     * @description Method finds a sprint given a sprint number
+     * @param {Number} sprintNo
+     * @returns {Object} the sprint object related to the given number
+     */
     getSprintByNumber(sprintNo) {
       return _.find(this.project.sprints, {
         sprintNumber: sprintNo.toString()
       });
     },
 
+    /**
+     * @description Methods returns the id of the given sprint
+     * @param {Object} sprint
+     * @returns {String} the id/key of the given sprint
+     */
     getSprintId(sprint) {
       return _.findKey(this.project.sprints, sprint);
     },
 
+    /**
+     * @description Method find and returns the most recent sprint
+     * @returns {Object} the most recent sprint
+     */
     getLatestSprint() {
       const sprintNumbers = _.map(this.project.sprints, sprint =>
         Number(sprint.sprintNumber)
```

Comment:
Should we be saving the sprintNumber as a string? As you are probably aware if we wasn't we could be doing:
```
const sprintNumbers = _.map(this.project.sprints, 'sprintNumber');
```
It seems we are converting the values to get the sprintNumber and later converting values to use them.

Is there a reason we can't store it as a Number?

Reply:
I'm sure there was a reason why I had to do it this way. I think there was some weirdness in the way it was treating the sprintNumber as a string/number when we didn't want it to... I can't really remember the specifics though haha